lib/ansible/modules/system/cgroup.py

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Cédric Nisio <cedric.nisio@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: cgroup
+
+short_description: Manage cgroup configurations
+
+version_added: "2.6"
+
+description:
+    - Manages cgroup configurations in the /etc/cgconfig.d directory.
```

Comment:
Use [formatting function](http://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#formatting-functions) `C()` for path and program name.